hackathon_site/event/jinja2/event/signup.html

```diff
@@ -0,0 +1,43 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Sign up{% endblock %}
+{% block form_button %}Sign up{% endblock %}
+
+{% block form_img %}<img src="{{ static('event/images/landing.svg') }}" />{% endblock %}
+               
+{% block form_inputs %}
+    <div class="input-field col s12">
+        <input id="first_name" type="text" class="validate">
+        <label for="first_name">first name</label>
```

Comment:
Should all these labels be capitalized (`First Name` or `First name`, etc)? Looks strange to me with them all being in lowercase